src/inc/til/winrt.h

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+namespace til // Terminal Implementation Library. Also: "Today I Learned"

```

Comment:
Do we need this in this PR? Curious, what for?

Reply:
Ah, I totally forgot I branched this off that dummy branch. We so don't. This was something I was fiddling with as a replacement for `WINRT_PROPERTY` but totally forgot about. I can revert these bits